pkg/commands/build/build.go

```diff
@@ -29,20 +29,12 @@ import (
 	"sigs.k8s.io/kustomize/pkg/target"
 )
 
-// BuildOptions contain the options for running a build
-type BuildOptions struct {
+// Options contain the options for running a build
+type Options struct {
 	kustomizationPath string
 	outputPath        string
 }
 
-// NewBuildOptions creates a BuildOptions object
-func NewBuildOptions(p, o string) *BuildOptions {
```

Comment:
Please don't remove this function. We need it in kubectl to create a `BuildOptions` object. 
```
o := NewBuildOptions(path, "")
o.RunBuild()
```

Reply:
ah yeah.

 can we call it build.Options instead of build.BuildOptions?

Reply:
Yes, go for it.

Reply:
To clear the linter errors.

Reply:
oky doky, done